spring-boot-actuator/pom.xml

```diff
@@ -139,6 +139,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.infinispan</groupId>
+			<artifactId>infinispan-core</artifactId>
+			<optional>true</optional>
+		</dependency>
```

Comment:
Why is that extra dependency necessary. Has something changed in 9.x that the core isn't provided with the embedded module?